apps/sms/views/conversation/js/conversation.js

```diff
@@ -94,6 +94,10 @@ var ConversationView = {
 
   multiSimActionButton: null,
   init: function conv_init() {
+    const THROTTLE_OPTS = {
```

Comment:
You can put this const at the top of the script, or treat is as conversationView's property like others.